Jenkinsfile-pr

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env groovy
+
+def lib = evaluate readTrusted('./jenkins.groovy')
+
+pipeline {
+    agent {
+        node {
+            label 'strimzi-pr'
+        }
+    }
+    parameters {
+        //execute all tests as default for each new Pull request
+        string(name: 'JUNIT_TAG', defaultValue: '', description: 'maven parameter for executing specific tests')
+    }
+    options {
+        timeout(time: 7, unit: 'HOURS')
+        ansiColor('xterm')
+    }
+    environment {
+        DOCKER_ORG="strimzi"
+        DOCKER_REGISTRY="docker.io"
+        DOCKER_TAG="latest"
```

Comment:
we can use there some specific ID like BUILD_ID from jenkins, but images are not pushed anywhere so that's the reason why we can use latest tag and DOCKER_ORG and DOCKER_REGISTRY with these values.

Reply:
If it's a PR job, we can pull images which were built by Travis. But again, DOCKER_TAG="latest" I guess can be related only for nightly job

Reply:
I will change it because it's confusing I guess. However - `If it's a PR job, we can pull images which were built by Travis` - are you sure that you want to use images from unstable build? how do you want to achieve that jenkins will wait for travis when finish? And as far as I can see from travis log, PR builds use localhost repos for store images.  `DOCKER_TAG="latest" I guess can be related only for nightly job` - that's correct, but this job does not  download any images from outside, all images are ready in local docker repo after build so it can use any tag and we don't need to push it anywhere. I see it's confusing so I will change it to something more reasonable.